waspc/data/Generator/templates/server/src/email/core/helpers.ts

```diff
@@ -0,0 +1,25 @@
+{{={= =}=}}
+
+import { EmailFromField } from "./types";
+
+export function createFromEmailString({
+  email,
+  title,
+}: {
+  email: string;
+  title?: string;
+}): string {
+  if (title) {
+    return `${title} <${email}>`;
+  }
+  return email;
+}
+
+export function getDefaultFromField(): EmailFromField {
```

Comment:
User must setup the default sender, but it he can override it when using the `emailSender.send({ ... })`

Reply:
> I thought defaultFrom was optional? Do you mean if they set it, then they must also set email?
This comment is outdated, sorry about that. `defaultFrom` is optional.

I'll check out the error 👍

Reply:
The error was caused by me sending `Nothing` as a value to the templates (instead of falling back to empty strings for example) and we can't do that. I've updated the code to used `fromMaybe "" ...` 

https://github.com/wasp-lang/wasp/pull/1050/commits/bd9070262216de3c0c9048f87a42e4f87373d909